tests/pyfunc/test_chat_model.py

```diff
@@ -84,6 +100,30 @@ def test_chat_model_save_throws_with_signature(tmp_path):
         )
 
 
+def mock_predict():
+    return "hello"
+
+
+def test_chat_model_with_context_saves_successfully(tmp_path_factory):
```

Comment:
```suggestion
def test_chat_model_with_context_saves_successfully(tmp_path):
```

can we use `tmp_path`?

Reply:
cause i'm using one for saving the model, and also the context artifact

Reply:
@daniellok-db 

Example:

```
d = tmp_path / "dir"
d.mkdir()
p = tmp_path / "file"
```

Reply:
🤯